internal/k8s/entity.go

```diff
@@ -395,3 +395,22 @@ func (e K8sEntity) HasKind(kind string) bool {
 	// TODO(maia): support kind aliases (e.g. "po" for "pod")
 	return strings.ToLower(e.GVK().Kind) == strings.ToLower(kind)
 }
+
+func NewNamespaceEntity(name string) K8sEntity {
+	yaml := fmt.Sprintf(`apiVersion: v1
```

Comment:
this is fine for now, but i'm a bit worried about the case where you have an existing namespace that gets overridden with different values

Reply:
oh hmm. Yeah good point. (you mean an existing namespace of `name` that also has like, some other metadata?)

added a commit that addresses this, tho if it's getting too finicky i can leave that safeguard off. wdyt?

Reply:
ya, exactly. i'm not totally sure what will happen if you overwrite someone's namespace with different metadata

i believe Helm checks the namespace before creating it
https://github.com/helm/helm/blob/release-2.15/pkg/kube/namespace.go#L43

Reply:
and fwiw, the issue isn't that you're overwriting the namespace in the chart, the issue is that you're overwriting some previously-installed namespace in the cluster

Reply:
that makes a lot more sense yeah. I guess I'll leave the safeguard against overwriting a namespace in your chart in, but ugh, that's more complicated...